test/functional/sql-tag/sql-tag-parameters-postgres.ts

```diff
@@ -0,0 +1,246 @@
+                    { id: "test3", value: 30, name: "c" },
+                ])
+
+                const minValue = 15
+                const maxValue = 25
+                const namePattern = "b"
+                const [example] = await connection.sql`SELECT * FROM example
+                    WHERE value > ${minValue}
+                    AND value < ${maxValue}
+                    AND name LIKE ${namePattern}
+                    ORDER BY value DESC`
+
+                expect(example.id).to.equal("test2")
+                expect(example.value).to.equal(20)
+            }),
+        ))
+
+    it("should handle SQL tag parameters with NULL values", () =>
+        Promise.all(
+            connections.map(async (connection) => {
+                const repo = connection.getRepository(PostgresExample)
+
+                await repo.save([
+                    { id: "null1", value: null },
+                    { id: "null2", value: 10 },
+                ])
+
+                const examples = await connection.sql`
+                    SELECT * FROM example WHERE value IS ${null}
+                `
+
+                const ids = examples.map((e: PostgresExample) => e.id)
+
+                expect(examples).to.have.length(1)
+                expect(ids).to.have.members(["null1"])
+            }),
+        ))
+
+    it("should handle SQL tag parameters with boolean values", () =>
+        Promise.all(
+            connections.map(async (connection) => {
+                const repo = connection.getRepository(PostgresExample)
+
+                await repo.save([
+                    { id: "true1", active: true },
+                    { id: "false1", active: false },
+                ])
+
+                const examples = await connection.sql`
+                    SELECT * FROM example WHERE active = ${true}
+                `
+
+                const ids = examples.map((e: PostgresExample) => e.id)
+
+                expect(examples).to.have.length(1)
+                expect(ids).to.have.members(["true1"])
+            }),
+        ))
+
+    it("should handle SQL tag parameters with date values", () =>
+        Promise.all(
+            connections.map(async (connection) => {
+                const repo = connection.getRepository(PostgresExample)
+                const now = new Date()
+                const yesterday = new Date(now.getTime() - 24 * 60 * 60 * 1000)
+
+                await repo.save([
+                    { id: "today", createdAt: now },
+                    { id: "yesterday", createdAt: yesterday },
+                ])
+
+                const examples = await connection.sql`
+                    SELECT * FROM example WHERE "createdAt" > ${yesterday}
+                `
+
+                const ids = examples.map((e: PostgresExample) => e.id)
+
+                expect(examples).to.have.length(1)
+                expect(ids).to.have.members(["today"])
+            }),
+        ))
+
```

Comment:
Likewise, although this test is actually binding a Date object to the parameter, it's only proving how the driver handles such values, not anything about the `sql` tag?
```suggestion
```